### ISSUE TITLE
[anipy-api] gogoanimeprovider crashes if there are no alternative names.

### ISSUE THREAD

**Issue by quenzt** [sdaqo/anipy-cli#192]
**Describe the bug**
Executing `info = anime.get_info()` in the anipy-api on an anime that does not have alternative names, makes the api crash with a NoneType exeption.

**To Reproduce**
this mini script demonstrates the issue
```
from anipy_api.provider import list_providers, get_provider 
from anipy_api.anime import Anime
from anipy_api.provider import LanguageTypeEnum

provider = get_provider("gogoanime")
supported_lang = {LanguageTypeEnum.SUB: True, LanguageTypeEnum.DUB: True}
anime = Anime(provider, "Beastars", "beastars", supported_lang)
info = anime.get_info()
```

**Expected behavior**
It should not crash. Rather just not return any alternative names.

**Info**
The error produced:
```
>>> info = anime.get_info()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/quenzt/newdl/anipy/lib/python3.12/site-packages/anipy_api/anime.py", line 92, in get_info
    return self.provider.get_info(self.identifier)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/quenzt/newdl/anipy/lib/python3.12/site-packages/anipy_api/provider/providers/gogo_provider.py", line 205, in get_info
    alternative_names = info_body.find("p", {"class": "other-name"}).find("a").text.split(",")  # type: ignore
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'find'
```
The bug happens on the latest release:
```
Name: anipy-api
Version: 3.1.10
Summary: api for anipy-cli
Home-page: https://sdaqo.github.io/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: /home/quenzt/newdl/anipy/lib/python3.12/site-packages
Requires: beautifulsoup4, dataclasses-json, levenshtein, m3u8, pycryptodomex, python-ffmpeg, python-mpv, requests
Required-by: anipy-cli
```

**Platform**
 - OS: linux (although i doubt thats relevant here)

**Additional context**
the code above demonstrates this bug for the anime "beastars" but any without additional names seems to cause the same issue.

**Comment by sdaqo** [comment]
Thanks for the report, will look into it!

**Comment by sdaqo** [comment]
fixed in 3.1.11
